CDTDatastore/HTTP/CDTHTTPInterceptorContext.m

```diff
@@ -19,16 +19,18 @@
 
 @implementation CDTHTTPInterceptorContext
 
+NSMutableDictionary<NSString*, NSObject*> *_state;
```

Comment:
Ah just watched the video a bit further on the job my memory, nothing os synthesised so we'd still need to define storage